### DISCUSSION TITLE
3ra Sesión de Requisitado

### DISCUSSION THREAD

**Discussion by natiicruuz** [natiicruuz/24-25-IdSw1-SDR#14] [Announcements]
Buenos días @paucres 

Como conversamos en la sesión de la semana pasada, quedábamos pendientes de ver si teníamos la 3.ª sesión esta semana o la que sigue.

La haremos la que sigue para poder brindarle un mayor avance en el proyecto junto con las recomendaciones de Manuel. 

Por lo mismo, proponemos una reunión de revisión de los avances el jueves a de 12:15 a 13:15 con @mmasias 

@MikaOT y yo estamos confirmados para ese día, el resto de compañeras nos confirmarán si podrán atender o no más adelante. 

Un saludo.

**Comment by natiicruuz** [comment]
@mmasias 

Llegamos a las 12:15 y Vimos que tenías una reunión hasta las 13:00 así que podemos re agendar para mañana? O si o podríamos generar un issue y hablarlo por ahí

**Reply by natiicruuz** [reply]
@mmasias 

Mañana a las 10:15 en el horario de clases manuel se puede reunir con nosotros para revisar y solventar las dudas que tengamos.

@BrendaLopes @sara-patino @MikaOT confirman si podrán asistir.

**Comment by mmasias** [comment]
Sí!

Disculpar por haberme saltado la reunión de hoy :(
-
Manuel Masías
http://es.linkedin.com/in/mmasias

**Comment by natiicruuz** [comment]
@mmasias podrías reservarnos un salón para la reunión de mañana a las 10:00?

**Comment by mmasias** [comment]
1.7, 10AM
